src/core/cff_parser.js

```diff
@@ -673,6 +679,9 @@ class CFFParser {
         }
       }
     }
+    if (length !== data.length) {
+      data[length] = 14;
```

Comment:
Unless I'm missing something obvious, this only works as long as *a single* `9` operator is found.
For the general case, I believe that you actually need something like:
```suggestion
    if (length < data.length) {
      data.fill(/* endchar = */ 14, length);
```

Reply:
I thought that the stuff after 14 would be ignored it's why I added only one 14.

Reply:
> I thought that the stuff after 14 would be ignored it's why I added only one 14.

That does sound reasonable; however I wonder if it's overall safer to have multiple `endchar`s rather than potentially unused "garbage" data at the end of a glyph?
*Please note:* I have no idea if this really matters, I'm just asking.